configuration.yaml

```
@@ -1,16 +1,15 @@
 global:
   logDir: gs://otar000-evidence_input/parser_logs
   cacheDir: cache_dir
-  # This is a directory prefix; individual schemas relative to it are specified by each rule.
   schema: https://raw.githubusercontent.com/opentargets/json_schema/2.2.7
-  EFOVersion: v3.40.0
+  EFOVersion: v3.42.0
 
 cancerBiomarkers:
   inputBucket: gs://otar000-evidence_input/CancerBiomarkers/data_files
   outputBucket: gs://otar000-evidence_input/CancerBiomarkers/json
   drugIndex: gs://open-targets-pre-data-releases/22.04/output/etl/parquet/molecule
 ChEMBL:
-  evidence: gs://otar008-chembl/cttv008-08-04-2022.json.gz
+  evidence: gs://otar008-chembl/cttv008-06-06-2022.json.gz
   stopReasonCategories: gs://otar000-evidence_input/ChEMBL/data_files/chembl_predictions-2022-04-08.tsv
```

```suggestion
  stopReasonCategories: gs://otar000-evidence_input/ChEMBL/data_files/chembl_predictions-2022-06-11.tsv
```

The predictions file needs to be updated to: `gs://otar000-evidence_input/ChEMBL/data_files/chembl_predictions-2022-06-11.tsv`